x-pack/plugins/maps/public/classes/layers/tiled_vector_layer/tiled_vector_layer.tsx

```diff
@@ -171,4 +197,20 @@ export class TiledVectorLayer extends VectorLayer {
     // higher resolution vector tiles cannot be displayed at lower-res
     return Math.max(this._source.getMinZoom(), super.getMinZoom());
   }
+
+  getFeatureById(
+    id: string | number | undefined,
+    meta: { mbProperties: GeoJsonProperties }
+  ): Feature | null {
+    const properties = this._source.getFeatureProperties(id, meta.mbProperties);
```

Comment:
I am not sure I follow the point of `_source.getFeatureProperties`. The implementation just returns `meta.mbProperties`. Is this function call to source even needed. Why not just use `meta.mbProperties` directly until different implementations are needed.

Reply:
see other comment. Removing this cruft.